packages/select/src/index.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react"
+import {useDefaults} from "@smashing/theme"
+import {SelectProps, OptionProps, Option} from "./types"
+import {Text} from "@smashing/typography"
+import {S, CheckIcon, ArrowIcon} from "./styles"
+import {Popover} from "@smashing/popover"
+
+const OptionItem: React.FC<OptionProps> = ({item, onClick, isActive}) => (
+  <S.SelectListItem onClick={onClick} isActive={isActive}>
+    {isActive && <CheckIcon />}
```

Comment:
There should be empty space for non selected values 
https://i.imgur.com/wzzyO13.png